utils/user-product-helper.ts

```diff
@@ -23,7 +23,7 @@ export function getProductInfo (config, type: ProductCode): ProductI {
     case ProductCode.PPR:
       return {
         image: 'PPR_dashboard_thumbnail_image.jpg',
-        link: config?.pprURL || 'link_not_configured',
```

Comment:
This old link already redirected to the PPR dashboard, but the new link is more direct.